corehq/apps/style/static/dashboard/less/dashboard.less

```diff
@@ -52,6 +52,7 @@
   .panel-heading {
     font-weight: bold;
     font-size: @font-size-large;
+    text-align: center;
```

Comment:
please create an extra style for center aligned headings. In general i'm not a fan of center aligned text. If a pop over is being cut off, have it float to the right or left.